### PR TITLE
[Update] Update version file

### PR DESCRIPTION
**Pull request by nbei**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -105,7 +105,7 @@ venv.bak/
 .mypy_cache/
 
 # custom
-mmedit/version.py
+# mmedit/version.py
```
Comment (hellock) [inline .gitignore:108]: Just remove this line

File: mmedit/__init__.py [hunk 2]
```diff
@@ -1 +1,30 @@
-from .version import __version__, short_version  # noqa: F401
+import mmcv
+
+from .version import __version__, version_info
+
+MMCV_MIN = '1.0.2'
+MMCV_MAX = '1.2.0'
```
Comment (hellock) [inline mmedit/__init__.py:6]: 1.2

File: setup.py [hunk 3]
```diff
@@ -54,26 +53,6 @@ def get_hash():
     return sha
```
Comment (hejm37) [inline setup.py:53]: Looks like the `get_hash` and `get_git_hash` are no longer used?

Comment (nbei) [reply setup.py:53]: I think this func may be called by extra repo. ^_^

**Review by hellock — APPROVED** [review]

**Pull request merged**